include/mata/nfa/delta.hh

```diff
@@ -179,6 +180,44 @@ public:
     Moves moves() const { return { .begin_ = moves_begin(), .end_ = moves_end() }; }
 }; // struct Post.
 
+/**
+ * @brief Specialization of Util::SynchronizedExistentialIterator for iterating over SymbolPosts.
+ */
+class SynchronizedExistentialIteratorSymbolPost : public Util::SynchronizedExistentialIterator<Util::OrdVector<SymbolPost>::const_iterator> {
+public:
+    /**
+     * @brief Get union of all targets.
+     */
+    StateSet unify_targets() const {
+        StateSet bigger_succ = {};
+        if(!this->is_synchronized()) {
+            return bigger_succ;
```

Comment:
I would refactor this so that `bigger_succ` is defined only after this if statement, and here we return `{}`. This explicitly shows that when `this` is not synchronized, we return empty set.

Reply:
ok, fixed